python/step_5.html

```diff
@@ -0,0 +1,27 @@
+<div class="instructions">
+    <h2>Our first method</h2>
+    <p>
+        We are not going to add the first method with some basic logic. We start simple, with the <em>ping</em> endpoint.
+        The desired behaviour is to receive a call from the pokerwars server, log it, and reply with a <em>pong</em>.
+    </p>
+    <p>
+        First we need to create a method linked to the <em>ping</em> endpoint. The syntax is <em>def</em>
```

Comment:
instead of describing the code which sounds difficult, why not put an example of the code and then explain it?

Reply:
I've done a bit of a mix now, should be easier